Compiler/Template/SimCodeTV.mo

```diff
@@ -395,6 +395,7 @@ package SimCode
       PartitionData partitionData;
       Option<DaeModeData> daeModeData;
       list<SimEqSystem> inlineEquations;
+      Option<OMSIData> omsiData;
```

Comment:
Is there redundant data that will be removed in favour of `omsiData`?

Reply:
Definitely. My guess would be, that most (better all) of the `list<SimEqSystem>` above are redundant and should be deleted in a working version of the OMSI.